translations/en.json

```diff
@@ -281,7 +280,6 @@
     "currency-display-name-mzn": "Mozambique Metical",
     "block": "Block",
     "wallet-set-up-title": "Set up your wallet",
-    "loading": "Loading... ",
```

Comment:
btw this "Loading... " was intentionally made to have "space" at the end to separate it out visually from the spinner. The spacing will probably need to be handled differently(:margin/:padding) where it is used.

Reply:
yes, I think it is better to manage it via styles instead of relying on a space.. but if you want i can leave the entry with the space and delete the other one...